Library/Homebrew/test/dev-cmd/prof_spec.rb

```diff
@@ -5,4 +5,24 @@
 
 describe "brew prof" do
   it_behaves_like "parseable arguments"
+
+  describe "integration tests", :integration_test do
```

Comment:
```suggestion
  describe "integration tests", :integration_test, :needs_network do
```

as it's doing gem installation, right?

Reply:
Good point!